runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -220,6 +229,36 @@ private long nonCachedAppend(Object object, UUID ... streamIDs) {
         return sv.append(object, null, CacheOption.WRITE_AROUND, streamIDs);
     }
 
+    /**
+     * @param partition the set of values to be written.
+     * @param num the current batch number
+     * @return
+     */
+    void writeCKJob(List<Map.Entry> partition, int num) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4660664)